device/accton/x86_64-accton_as7726_32x-r0/pddf/pd-plugin.json

```diff
@@ -0,0 +1,35 @@
+{
+	"PSU":
```

Comment:
Please use four spaces for indenting JSON files in lieu of tabs.

Reply:
Will correct it and file a new review with PDDF generic module changes only. Generic plugins will be pushed with 2.0 APIs

Reply:
We do have plan to move PDDF generic plugins to 2.0 APIs. Tentatively, we will complete movement to 2.0 by November 2019. 

Currently, these plugins are based on 1.0 with some enhancements in base classes. The enhancements in base classes are part of the other review, https://github.com/Azure/sonic-platform-common/pull/62/

PDDF support has been added for couple of platforms and that is the reason we want to push these changes.